source/scenario_gen_proximity.rst

```diff
@@ -142,112 +142,62 @@ generator is a base land-use/land-cover map and user-defined land cover
-       scenario, convertible land covers and habitat of interest
-       land-covers may be the same, or a subset of one another, or they
-       can be different.
-
-8. Number of Steps in Conversion: enter an integer for the number of
-   steps the simulation should take to fragment the habitat of interest
-   in the fragmentation scenario. Entering a 1 means that all of the
-   habitat conversion will occur in the center of the patch of the
-   habitat of interest. Entering 10 will be fragmented according to a
-   pattern of sequentially converting the pixels furthest from the edge
-   of that habitat, over the number of steps specified by the user.
+- :investspec:`scenario_gen_proximity workspace_dir`
+
+- :investspec:`scenario_gen_proximity results_suffix`
+
+- :investspec:`scenario_gen_proximity base_lulc_path`
+
+- :investspec:`scenario_gen_proximity aoi_path` Prior to scenario generation, the map will be clipped to the extent of this vector.
+
+- :investspec:`scenario_gen_proximity area_to_convert` As many pixels as possible will be converted without exceeding this area.
+
+- :investspec:`scenario_gen_proximity focal_landcover_codes`
+
+- :investspec:`scenario_gen_proximity convertible_landcover_codes`
+
+- :investspec:`scenario_gen_proximity replacment_lucode` If there are multiple LULC types that are of interest for conversion, this tool should be run in sequence, choosing one type of conversion each time. A new code may be introduced if it is a novel land-use for the region or if it is desirable to track the expanded land-use as separate from historic land-use.
+
+- :investspec:`scenario_gen_proximity convert_farthest_from_edge` Convertible land covers and habitat of interest land-covers may be the same, or a subset of one another, or they can be different. If they are the same, the number of steps for conversion should be specified, because the conversion of habitat within the focal land cover will create new habitat edge, resulting in a completely different pattern of conversion depending on how many steps are chosen.
+
+- :investspec:`scenario_gen_proximity convert_nearest_to_edge` Convertible land covers and habitat of interest land-covers may be the same, or a subset of one another, or they can be different.
+
+- :investspec:`scenario_gen_proximity n_fragmentation_steps` Entering a 1 means that all of the habitat conversion will occur in the center of the patch of the habitat of interest. Entering 10 will be fragmented according to a pattern of sequentially converting the pixels furthest from the edge of that habitat, over the number of steps specified by the user.
 
 Interpreting Results
 --------------------
 
 Final Results
 ~~~~~~~~~~~~~
 
--  **InVEST….log…txt:** Each time the model is run, a text (.txt) file
-   will appear in the \ *Output* folder. The file will list the
+- **InVEST….log…txt:** Each time the model is run, a text (.txt) file
+   will appear in the *Output* folder. The file will list the

```

Comment:
Looking at the html there's a newline at the end of ` a text (.txt) file` that breaks up the text. It's not a big deal but occurs for the rest of the Final and Intermediate results.

Reply:
good catch!